utils/testutils/testutils.go

```diff
@@ -72,3 +75,36 @@ func GetInterface(port int) string {
 	}
 	return string(out)
 }
+
+func SetReplicationLagForTesting(ss *sqlstore.SqlStore, seconds int) error {
```

Comment:
That's not quite what I meant, but now that we have done this, it's not a big deal. :P

Reply:
What did you have in mind Agniva?

Reply:
Ah I see :)

Well, I wanted the method to be moved to a `_test.go` file under the same `sqlstore` package. So there would be no code changes from the callsite, but the method wouldn't appear in built binaries, and is semantically more clear.